released/packages/itauto/itauto.8.14.0/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "frederic.besson@inria.fr"
+homepage: "https://gitlab.inria.fr/fbesson/itauto"
+dev-repo: "git+https://gitlab.inria.fr/fbesson/itauto.git"
+authors: ["Frédéric Besson"]
+bug-reports: "frederic.besson@inria.fr"
+license: "GPL 3"
```

Comment:
@fajb can you clarify if this is `GPL-3.0-only` or `GPL-3.0-or-later`?

Also, are you aware of the incompatibility between GPL and CECILL B/C that could affect which projects can use itauto? https://www.gnu.org/licenses/license-list.html#CeCILL-B

Reply:
I did not give it much thoughts. Can't even remember how and/or why it ended up being GPL 3.

Reply:
In this case let me link our general license recommendations in coq-community: https://github.com/coq-community/manifesto#best-practices

But should we say `GPL-3.0-or-later` for now? You can always revisit license details in the next release (or revise the current one in a new PR).

Reply:
I changed for MIT